spec/system/page_objects/pages/review.rb

```diff
@@ -64,6 +64,12 @@ def has_reviewable_with_rejected_status?(reviewable)
         within(reviewable_by_id(reviewable.id)) { page.has_css?(".status .rejected") }
       end
 
+      def has_reviewable_with_rejection_reason?(reviewable, rejection_reason)
+        within(reviewable_by_id(reviewable.id)) do
```

Comment:
Try not to use `within`, it has been super janky for us in the past, and `reviewable_by_id(reviewable.id).has_css?(".reviewable-user-details.reject-reason .value", text: rejection_reason)` should work fine too (as long as `reviewable_by_id` is a `find` call)